README.rst

```diff
@@ -2,4 +2,28 @@
 lsst.validate.metrics
 #####################
 
-**Centralized metric and specification definitions for the ``lsst.validate`` framework.**
+**Centralized metric and specification definitions for the lsst.validate framework.**
```

Comment:
I pushed some new README content. Let me know if those are to your liking. I don't want to document too much to early since we're still prototyping ideas in sqr-017.lsst.io.